src/components/__tests__/Discipline-test.js

```diff
@@ -0,0 +1,16 @@
+import 'react-native'
+import React from 'react'
+import renderer from 'react-test-renderer'
+import Discipline from '../Discipline'
+
+it('renders with a title', () => {
+  const tree = renderer.create(
+    <Discipline
+      title='O Hai!'
+      icon='arts'
+      tag='test'
+      color='#fff'
+      setSelectedProjectTag={jest.fn} />
+  ).toJSON()
+  expect(tree).toMatchSnapshot()
```

Comment:
In a nutshell, these snapshots are testing that the component, `Discipline` in this case, is rendering as expected based on the props passed in. Is that a correct assumption of these snapshots?